Source/WarpX.cpp

```diff
@@ -557,6 +557,10 @@ WarpX::ReadParameters ()
         pp.query("pml_has_particles", pml_has_particles);
         pp.query("do_pml_j_damping", do_pml_j_damping);
         pp.query("do_pml_in_domain", do_pml_in_domain);
+#ifdef WARPX_DIM_RZ
+        AMREX_ALWAYS_ASSERT_WITH_MESSAGE( do_pml==1,
```

Comment:
Shouldn't this be `do_pml==0`?

Reply:
Oops, yes indeed! I just corrected it. Thanks for catching this!